docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -1,4 +1,128 @@
-# Deploy to Fly.io with SQLite
+# Deploy Platformatic Applications to Fly.io
+
+## Deploying a Platformatic Runtime Application 
+
+This guide provides instructions on deploying a Platformatic Runtime application to Fly.io. With a runtime application, you are deploying your entire application, including all services in the `services` folder.
+
+
+### Dockerfile for Runtime Application
+
+Here is an example Dockerfile for a Platformatic Runtime application:
+
+```dockerfile
+FROM node:20-alpine
+
+ARG VITE_AI_URL
+ENV VITE_AI_URL=$VITE_AI_URL
+
+ENV APP_HOME=/home/app/node/
+
+WORKDIR $APP_HOME
+
+COPY package.json package.json
+COPY package-lock.json package-lock.json
```

Comment:
You want to do `npm ci` after this, so that you save the time of `npm install` during rebuilds

Reply:
Curious, will this also be copy and install the packages?

Reply:
ah no, we should skip those with a `.dockerignore` file.